### ISSUE TITLE
Feature request: Add hclfmt option for only logging names of files which have been modified/formatted

### ISSUE THREAD

**Issue by a1031** [gruntwork-io/terragrunt#1824]
The default behavior of [terraform fmt](https://www.terraform.io/docs/cli/commands/fmt.html) is to only output the names of files which have been modified/formatted, but the default behavior of `terragrunt hclfmt` is to output the paths of all files checked (at [terragrunt-log-level](https://terragrunt.gruntwork.io/docs/reference/cli-options/#terragrunt-log-level) `info`). The latter can end up outputting a very long list to the screen for larger projects (especially since there's usually one `.terraform.lock.hcl` file for each `terragrunt.hcl` file, resulting in double the number of lines of output).

This is fine when used in a CI pipeline, but for when running locally it would be nice to have an option to only output the paths of files which have been modified.

I've tried using different log levels and also checked [cli/hclfmt.go](https://github.com/gruntwork-io/terragrunt/blob/fd49e82778353b2fe5bc52e46ef2c5a5130374c9/cli/hclfmt.go) to see if this could be accomplished using `--terragrunt-log-level` but it doesn't seem to be possible.

I started writing my own bash script to accomplish this by:
- Running `terragrunt hclfmt --terragrunt-check` in a `while` loop with stdout and stderr captured in a variable or piped to another command
- If a file with an invalid format is encountered:
  - Grepping in the variable (or stdin) for `Invalid file format` and using `sed` to extract the file's path
  - Running `terragrunt hclfmt --terragrunt-hclfmt-file <path>` to format any offending files individually

But I can't figure out a way to capture the output of `terragrunt hclfmt --terragrunt-check`. If I run `terragrunt hclfmt --terragrunt-check | grep Invalid` I still see all of the output. If I run `test="$(terragrunt hclfmt --terragrunt-check)" && echo $test` the variable is empty. So all I can do now is determine whether the check errored out, but can't identify the file which caused it to error.

**Comment by yorinasub17** [comment]
Makes sense to modify the logging so that files that are checked are logged at the DEBUG level, and files that are modified are logged at the INFO level to accomplish this. A PR to implement that would be most appreciated!

**Comment by yorinasub17** [comment]
The solution would be to demote these logs to `DEBUG` level, and then add new logs at `INFO` level that will show which files were formatted.

E.g.:

```
DEBUG[0000] Formatting hcl files from the directory tree source-dir/my-code-dir.
DEBUG[0000] Formatting source-dir/my-code-dir/terragrunt.hcl
INFO[0000] source-dir/my-code-dir/terragrunt.hcl was updated.
DEBUG[0000] Formatting source-dir/my-code-dir/.terraform.lock.hcl
DEBUG[0000] Formatting source-dir/my-code-dir/deploy/terragrunt.hcl
INFO[0000] source-dir/my-code-dir/deploy/terragrunt.hcl was updated.
DEBUG[0000] Formatting source-dir/my-code-dir/deploy/terragrunt.lock.hcl
```

That way, when the log level is `INFO` (default) or higher, only the `was updated` logs will show.

The relevant code is here: https://github.com/gruntwork-io/terragrunt/blob/master/cli/hclfmt.go#L32
